src/index.html

```diff
@@ -7,6 +7,381 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#home" class="header__link is-active">Apple</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Samsung</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Smartphones</a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Gadgets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Tablets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Photo</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page">
+      <section class="page__cards cards">
+        <article class="cards__item card" data-qa="card">
+
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+
+          <h2 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__reviews">
+            <div class="stars">
+              <div class="stars__img"></div>
+              <div class="stars__img"></div>
+              <div class="stars__img"></div>
+              <div class="stars__img"></div>
+              <div class="stars__img"></div>
+            </div>
+
+            <div class="card__number-rewiews">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-count">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+
+        </article>
```

Comment:
Chek it everywhere, please
```suggestion
          </a>
        </article>
```